tembo-pgmq-python/tests/.env

```diff
@@ -0,0 +1,5 @@
+PG_HOST=foolishly-mellow-cottonmouth.data-1.use1.tembo.io
+PG_PORT=5432
+PG_USERNAME=postgres
+PG_PASSWORD=qNr4LAz4FAVzRNNj
+PG_DATABASE=postgres
```

Comment:
I'm guessing this was unintentionally added? Should probably use a local database instance for tests, etc

Reply:
Yes it was unintentional I'll fix it.